### ISSUE TITLE
A11y failures meaning my website isn't WCAG AA

### ISSUE THREAD

**Issue by adamjamesturner93** [disqus/disqus-react#143]
### Is your feature request related to a problem? Please describe.  
I really like disqus but the plugin isn't accessible and is preventing me from having a AA accessible website. 

### Describe the solution you'd like  
Updated colours and a11y tags to remove accessibility flags

### Describe alternatives you've considered  
None.

### Additional context  
![Screenshot 2023-01-10 at 16 25 09](https://user-images.githubusercontent.com/50718647/211606576-c8e63ab3-8a82-408a-8bba-f7a65dbd3ad5.png)
